semgrep_output_v1.atd

```diff
@@ -1113,29 +1114,35 @@ type scan_request = {
 }
 
 (* Response by the backend to the CLI to the POST /scans *)
-(* TODO
 type scan_response = {
-    scan: scan_response_bis;
+    scan: scan_info;
+    scan_config: scan_configuration;
```

Comment:
would be good to introduce here the new and more general 'ci_config_from_cloud'
that hopefully the Frontend and backend could start to use to send scan config info.